index.html

```diff
@@ -1,69 +1,269 @@
+              <i class="bar"></i>
+          </div>
+        </nav>
+    </header>
+
+    <main class="flex-container wrapper">  
+      <section id="head-container back_image section_white--bottom">
+        <h1 class="headers">
+          I'm Montserrat, <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience. If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me!
+        </p>
+        <h2 class="connect">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="https://twitter.com/montsecruzg9" rel="noopener noreferrer" target="_blank">
+              <img src="./assets/twitter.png" alt="twitter" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/montserrat-cruz-695594120/" rel="noopener noreferrer" target="_blank">
+              <img src="assets/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="assets/medium.png" alt="medium" />
+            </a>
+          </li>
+          <li>
+            <a href="https://github.com/montsecruz9" rel="noopener noreferrer" target="_blank">
+              <img src="assets/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="assets/angel.png" alt="angel" />
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="section" id="portfolio">
+        <div class="portfolio">
+          <article class="work_card">
+            <header class="work__header">
+              <img
+                src="assets/snapshoot1.png"
+                alt="Snapshoot Portfolio 1"
+                class="work__image"
+              />
+            </header>
+            <div class="portfolio__content">
+              <h3 class="portfolio__title color-dark">Tonic</h3>
+              <div class="portfolio__profile">
+                <h4 class="portfolio__author color-info">Canopy</h4>
+                <ul class="portfolio__list">
+                  <li class="work__item">Back End Dev</li>
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p class="primary_description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
```

Comment:
- [ ] Nice efforts here 👏 . But please remove the underline that you have in the button. The idea is to have a close replica of the design. 👍 

## Figma:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/39d4eb32-40bd-4cf5-a5e7-dd296e99f668)

## Your design:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/d4b37f72-aac0-4bba-99d2-c8dee8fb46bd)